app/models/financialDetails/CreditType.scala

```diff
@@ -53,8 +53,9 @@ object CreditType {
   private val mfaCredit = Range.inclusive(4004, 4025)
     .filterNot(_ == 4010).filterNot(_ == 4020).map(_.toString)
     .toList
-  // TODO: This should be "0060", not "4920" or "4930"
+  // TODO: MISUV-7759 - This should be "0060", not "4920" or "4930"
   private val payment = List("4920", "4930")
+  //  private val payment = List("0060")
```

Comment:
I would assume we can tidy up commented code here?

Reply:
It's for MISUV-7559 - I thought it better to get this ticket over the line, rather than carry it over because of the AT changes required